docs/fleet/machines.md

```diff
@@ -44,8 +44,10 @@ Select your system's architecture and select the version of the {{< glossary_too
 Then, follow the instructions on the page to connect and set up your machine.
 
 {{% alert title="Tip" color="tip" %}}
-The **Micro-RDK** is for [microcontrollers](/get-started/installation/prepare/microcontrollers/).
+If your machine is controlled by a microcontroller, install the [**Micro-RDK**](/get-started/installation/prepare/microcontrollers/) instead of full `viam-server`.
+{{% /alert %}}
 
+{{% alert Title="Tip" color ="tip" %}}
```

Comment:
does this one need to be an alert? I think one or the other should just be regular content.

Reply:
True I agree. okay